hedera-node/configuration/preprod/bootstrap.properties

```diff
@@ -3,3 +3,6 @@ contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
 hedera.recordStream.enableTraceabilityMigration=false
+autoCreation.enabled=true
+lazyCreation.enabled=false
```

Comment:
Actually for `preprod` we'd probably want `true` here and next line I think.

Reply:
Updated.